academy/4-my-own-chain/cosmjs-gui.md

```diff
@@ -905,22 +907,22 @@ With this done:
 
     <HighlightBox type="tip">
 
-    There is a potentially hard-to-reproduce-in-production **race condition** bug here. The `loadGame` is done right after the transaction has completed. However, depending on the implementation of the RPC end point, your `playGuiMove` and your `loadGame` calls may be hitting two different servers on the backend. In some instances, the server that answers your `loadGame` may not have fully updated its store and may in fact serve you the **old** version of your game.
+    There is a potentially hard-to-reproduce-in-production **race condition** bug here. The `loadGame` is done immediately after the transaction has completed. However, depending on the implementation of the RPC end point, the `playGuiMove` and `loadGame` calls may hit two different servers on the backend. In some instances, the server that answers your `loadGame` may not have fully updated its store and may in fact serve you the **old** version of your game.
 
-    So, as your GUI matures, you may want to show the _expected_ state of the game, before you eventually show its _finalized_ state. Sometimes you may want to show the expected state of the game even before the transaction has completed, and add visual cues hinting at the fact it is a **provisional** state.
+    Aas your GUI matures, you may want to show the _expected_ state of the game before you eventually show its _finalized_ state. Sometimes you may want to show the expected state of the game even before the transaction has completed, and add visual cues hinting at the fact it is a **provisional** state.
 
-    The same can happen when creating the game, where the second server may return `null` if it has not been updated yet.
+    The same can happen when creating a game, where the second server may return `null` if it has not been updated yet.
 
     </HighlightBox>
 
-Now you can try yourself in the GUI. Of course, make sure you put your Keplr address as the black player, so that you start with this one. Then either you have a second account on Keplr with which to play red, or you play red from the command line.
+Now you can play test in the GUI. Make sure to put your Keplr address as the black player, so that you start with this one. You will need a second account on Keplr with which to play red, otherwise you must play red from the command line.
```

Comment:
```suggestion
Now you can play test in the GUI. Make sure to put your Keplr address as the black player, so that you start with this one. You will need a second account on Keplr with which to play red, otherwise you must play red from the command line. Alternatively, put the same Keplr address for both black and red.
```